apps/webapp/src/components/header/header.tsx

```diff
@@ -1,17 +1,28 @@
-import { Navbar } from './navbar';
+import { LayoutLoaderResult } from '../layout.tsx';
 import { Link, useLoaderData } from 'react-router-dom';
-import { PagePath } from '../metadata/paths.ts';
+import { MessageWarningIcon } from '../../icons/message-warning.tsx';
+import { MobileNavMenu } from './mobile-nav-menu.tsx';
+import { Navbar } from './navbar';
+import {
+  NetworksPopover,
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@penumbra-zone/ui';
 import Notifications from './notifications.tsx';
+import { PagePath } from '../metadata/paths.ts';
 import { TabletNavMenu } from './tablet-nav-menu.tsx';
-import { LayoutLoaderResult } from '../layout.tsx';
-import { NetworksPopover } from '@penumbra-zone/ui';
-import { MobileNavMenu } from './mobile-nav-menu.tsx';
+
+// Infinite-expiry invite link, provided by Henry, to the #web-ext-feedback
+// channel.
+const WEB_EXT_FEEDBACK_DISCORD_CHANNEL = 'https://discord.gg/XDNcrhKVwV';
 
 export const Header = () => {
   const result = useLoaderData() as LayoutLoaderResult;
 
   return (
-    <header className='z-10 flex w-full flex-col items-center justify-between px-6 md:h-[82px] md:flex-row md:px-12'>
+    <header className='z-10 flex w-full flex-col items-center justify-between px-6 md:h-[82px] md:flex-row md:gap-12 md:px-12'>
```

Comment:
Added a gap so that the nav bar would have some space between it and the other elements on narrower screens